javascript-practice/src/js/views/dashboardView.js

```diff
@@ -78,18 +115,115 @@ export default class DashboardView {
       const valueAddPassword = this.addPasswordEl.value;
       const valueAddConfirmPassword = this.addConfirmPasswordEl.value;
 
-      submitAddUser(
-        valueAddUserId,
-        valueAddFirstNameId,
-        valueAddLastNameId,
-        valueAddEmailId,
-        valueAddMobileNoId,
-        valueAddRole,
-        valueAddUserName,
-        valueAddPassword,
-        valueAddConfirmPassword
-      );
+      Object.values(this.addUserErrorEls).forEach((el) => {
+        el.textContent = '';
+      });
+
+      if (
+        !valueAddUserId ||
+        !valueAddFirstNameId ||
+        !valueAddLastNameId ||
+        !valueAddEmailId ||
+        !valueAddMobileNoId ||
+        !valueAddUserName ||
+        !valueAddPassword ||
+        !valueAddConfirmPassword
+      ) {
+        Object.entries(this.addUserErrorEls).forEach(([key, el]) => {
+          if (!this[key].value) {
+            el.textContent = `${REQUIRED_FIELD}`;
+          }
+        });
+
+        return;
+      }
+      let isValid = true;
+      if (isValid) {
+        if (
+          !validateInputLength(valueAddUserId) ||
+          !validateInputLength(valueAddFirstNameId) ||
+          !validateInputLength(valueAddLastNameId) ||
+          !validateInputLength(valueAddUserName)
+        ) {
+          if (
+            !validateInputLength(valueAddUserId) &&
+            this.addUserErrorEls.addUserIdEl
+          )
+            this.addUserErrorEls.addUserIdEl.textContent = `${REQUIRED_FIELD_THREE}`;
+          if (
```

Comment:
Why do some field should have more than 3 letter? Like first name, last name?

Reply:
i done fix